test/jest/__snapshots__/l-gradient.jest.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`LGradient basic renders 1`] = `<noscript />`;
+
+exports[`LGradient opacity renders 1`] = `<noscript />`;
```

Comment:
These don't seem super informative?

Reply:
Ahh yeah ... at least one of these test cases should have an active prop passed. Thanks for the catch